versioned_docs/version-1.2/partials/_elemental-operator-install.md

```diff
@@ -5,6 +5,11 @@ cluster and taking care of creating inventories, registrations for machines and
 
 We will use the Helm package manager to install the elemental-operator chart into our cluster.
 
+```shell showLineNumbers
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator-crds oci://registry.suse.com/rancher/elemental-operator-crds-chart
+helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator oci://registry.suse.com/rancher/elemental-operator-chart
+```
```

Comment:
Shouldn't we target a specific chart version in the versioned docs (i.e., --version 1.2.5 parameter)? At least in the installation instruction, for the reference and troubleshooting we may leave the *unversioned* ones.